Jellyfin.Data/Entities/ActivityLog.cs

```diff
@@ -0,0 +1,153 @@
+            this.Name = name;
+
+            if (string.IsNullOrEmpty(type)) throw new ArgumentNullException(nameof(type));
+            this.Type = type;
+
+            this.UserId = userid;
+
+            this.DateCreated = datecreated;
+
+            this.LogSeverity = logseverity;
+
+
+            Init();
+        }
+
+        /// <summary>
+        /// Static create function (for use in LINQ queries, etc.)
+        /// </summary>
+        /// <param name="name"></param>
+        /// <param name="type"></param>
+        /// <param name="userid"></param>
+        /// <param name="datecreated"></param>
+        /// <param name="logseverity"></param>
+        public static ActivityLog Create(string name, string type, Guid userid, DateTime datecreated, Microsoft.Extensions.Logging.LogLevel logseverity)
+        {
+            return new ActivityLog(name, type, userid, datecreated, logseverity);
+        }
+
+        /*************************************************************************
+         * Properties
+         *************************************************************************/
+
+        /// <summary>
+        /// Identity, Indexed, Required
+        /// </summary>
+        [Key]
+        [Required]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public int Id { get; protected set; }
+
+        /// <summary>
+        /// Required, Max length = 512
+        /// </summary>
+        [Required]
+        [MaxLength(512)]
+        [StringLength(512)]
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Max length = 512
+        /// </summary>
+        [MaxLength(512)]
+        [StringLength(512)]
+        public string Overview { get; set; }
+
+        /// <summary>
+        /// Max length = 512
+        /// </summary>
+        [MaxLength(512)]
+        [StringLength(512)]
+        public string ShortOverview { get; set; }
+
+        /// <summary>
+        /// Required, Max length = 256
+        /// </summary>
+        [Required]
+        [MaxLength(256)]
+        [StringLength(256)]
+        public string Type { get; set; }
+
+        /// <summary>
+        /// Required
+        /// </summary>
+        [Required]
+        public Guid UserId { get; set; }
+
+        /// <summary>
+        /// Max length = 256
+        /// </summary>
+        [MaxLength(256)]
+        [StringLength(256)]
+        public string ItemId { get; set; }
+
+        /// <summary>
+        /// Required
+        /// </summary>
+        [Required]
+        public DateTime DateCreated { get; set; }
+
+        /// <summary>
+        /// Required
+        /// </summary>
+        [Required]
+        public Microsoft.Extensions.Logging.LogLevel LogSeverity { get; set; }
+
+        /// <summary>
+        /// Required, ConcurrenyToken
+        /// </summary>
+        [ConcurrencyCheck]
+        [Required]
+        public uint RowVersion { get; set; }
```

Comment:
Properties should have actual documentation

Reply:
I was planning on documenting all of the new entity classes in a separate PR, but if you think this is a blocker I can do it here

Reply:
Making a docs pass works for me!